Bottom: 102ef3e359d6e323538945887e09c42a2899d9be
Top:    b93855e1bd65068e4f74abc73fea8a267b8557f5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 01:58:53 +0100

Win32: Select the correct GL libraries to link against


---

diff --git a/configure.ac b/configure.ac
index 74d6e9e..cd696a2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -429,7 +429,7 @@ Either do not use --enable-gl or enable the gtk HID.])
 	esac
 
 	GLU_CFLAGS=
-	GLU_LIBS=-lGLU
+	GLU_LIBS=-lglu32 -lopengl32
 
 	AC_DEFINE([ENABLE_GL], 1,
 		[Define to 1 if GL support is to be compiled in])
